### PR TITLE
document Float#{clip,fold,wrap}

### PR DESCRIPTION
**Pull request by mxw**
## Purpose and Motivation

Documented methods are morally superior.

## Types of changes

- Documentation

## To-do list

- [x] Updated documentation
- [x] This PR is ready for review

### CODE REVIEW

**Comment by mxw** [comment]
@dyfer can do—is there a, uh, label that describes these three operations that would make an appropriate header in the Integer docs?

**Comment by redFrik** [comment]
i believe COPYMETHOD could be useful here.
put this in Integer.schelp...
```
copymethod:: Float -clip
copymethod:: Float -fold
copymethod:: Float -wrap
```

**Comment by dyfer** [comment]
Ah thanks @redFrik !

**Comment by dyfer** [comment]
Thanks again @mxw and sorry for putting you on the wrong track :)

**Review by dyfer — APPROVED** [review]

File: HelpSource/Classes/Integer.schelp [hunk 1]
```diff
@@ -214,6 +214,13 @@ code::
 ::
 
 
+subsection:: Signal Processing
```
[the remark this answers is not in the document]
Comment (mxw) [reply HelpSource/Classes/Integer.schelp:217]: feel free to change it to something more suitable. i've never encountered a need to clip a number in this way outside of signal processing applications, so i figured it was a good enough subsection name for now (and better than tossing them into a "miscellaneous" section).

**Comment by jamshark70** [comment]
Note that Float vs Integer .wrap treat the bounds differently.- Float wraps lo <= x < hi- Integer wraps lo <= x <= hiI believe fold and clip are both inclusive bounds on both sides, but new documentation for these methods should note this difference for wrap.

File: HelpSource/Classes/Integer.schelp (hunk 1 above)
[the remark this answers is not in the document]
Comment (telephon) [reply HelpSource/Classes/Integer.schelp:217]: yes, I see. Just a suggestion:

`subsection:: Range Limiting`

**Comment by mxw** [comment]
that is an extremely uncomfortable discrepancy, but at least it's true. see #5727.

**Comment by dyfer** [comment]
@paum3 changes pulled into develop will typically be available in the next SC _minor_ release (major.minor.bugfix), in this case 3.13.0. You can also get the "latest develop" build from the regular SC download page (at least on Windows and macOS).

**Pull request merged**